libcudacxx/test/support/test_execution_policies.h

```diff
@@ -0,0 +1,55 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef TEST_SUPPORT_TEST_EXECUTION_POLICIES
+#define TEST_SUPPORT_TEST_EXECUTION_POLICIES
+
+#include <cuda/std/cstdlib>
+#include <cuda/std/execution>
+#include <cuda/std/type_traits>
+#include <cuda/std/utility>
+
+#include "test_macros.h"
+
+#define EXECUTION_POLICY_SFINAE_TEST(function)                                                                          \
+  template <class, class...>                                                                                            \
+  struct sfinae_test_##function##_impl : cuda::std::true_type                                                           \
```

Comment:
suggestion: I think it'd match intuition better if test returned true on success:

```suggestion
  struct sfinae_test_##function##_impl : cuda::std::false_type                                                           \
```